tests/test_generate.py

```diff
@@ -277,6 +293,41 @@ def test_generate(yaml_dir, tmpdir):
     t7 = html_blob.updated
     assert t6 != t7
 
+    # Upload new blob, build only latest, and verify only latest is updated.
+    only_latest = True
+    new_metadata = "docs.metadata.newer"
+    swap_file(yaml_dir, yaml_dir / "docs.metadata", yaml_dir / new_metadata)
+    latest_html_blob_name = "python-doc-pipeline-test-2.1.2.tar.gz"
+
+    upload_yaml(yaml_dir, test_bucket)
+    generate.build_all_docs(test_bucket, storage_client, only_latest)
```

Comment:
Consider `only_latest=True` to avoid the redirection of defining `only_latest` as a separate variable. When you're reading just this line (or the one below), you don't know the value of `only_latest`.